e2e-tests/e2e-fixtures/gef/constants.js

```diff
@@ -0,0 +1,37 @@
+const {
+  PORTAL_USER_ROLES: USER_ROLES,
+  PORTAL_USER_SIGN_IN_TOKENS: SIGN_IN_TOKENS,
+  DEAL_TYPE,
+  SUBMISSION_TYPE: DEAL_SUBMISSION_TYPE,
+  DEAL_STATUS,
+  FACILITY_TYPE,
+} = require('../constants.fixture');
+
+const DEAL_COMMENT_TYPE_PORTAL = {
+  UKEF_COMMENT: 'ukefComments',
+  UKEF_DECISION: 'ukefDecision',
+};
+
+const PORTAL_ACTIVITY_LABEL = {
```

Comment:
Can we delete this please and instead, in the relevant tests, import this from `libs/common`